internal/install/recipe_installer.go

```diff
@@ -194,29 +227,36 @@ func (i *RecipeInstaller) installRecipesWithPrompts(m *types.DiscoveryManifest,
 		if i.RecipesProvided() || i.AssumeYes {
 			ok = true
 		} else {
+			log.Debugf("Checking user accepts install...")
 			ok, err = i.userAcceptsInstall(r)
 			if err != nil {
+				log.Debugf(fmt.Sprintf("Done installing recipes with prompts, exception:%s", err))
```

Comment:
fmt.Sprintf is unnecessary from within log.Debugf - you can use it the same way you would Sprintf.